tstypes/type/introspection.d.ts

```diff
@@ -40,6 +30,8 @@ export const SchemaMetaFieldDef: GraphQLField<any, any>;
 export const TypeMetaFieldDef: GraphQLField<any, any>;
 export const TypeNameMetaFieldDef: GraphQLField<any, any>;
 
-export const introspectionTypes: ReadonlyArray<any>;
+export const introspectionTypes: ReadonlyArray<
+  GraphQLObjectType | GraphQLEnumType
```

Comment:
I think it's better to make it just `GraphQLType` since we can latter add different types.

Reply:
Got ti